operators/aqua/6.5.5/aqua-operator.v6.5.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,574 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aqua-operator.v6.5.5
+  namespace: placeholder
+  annotations:
+    # Setting olm.maxOpenShiftVersion automatically
+    # This property was added via an automatic process since it was possible to identify that this distribution uses API(s),
+    # which will be removed in the k8s version 1.22 and OpenShift version OCP 4.9. Then, it will prevent OCP users to
+    # upgrade their cluster to 4.9 before they have installed in their current clusters a version of your operator that
+    # is compatible with it. Please, ensure that your project is no longer using these API(s) and that you start to
+    # distribute solutions which is compatible with Openshift 4.9.
+    # For further information, check the README of this repository.
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
```

Comment:
Hi @yossig-aquasec, 
 
Why are you adding the max ocp version == 4.10?
Also, please look at the top comment and see that it is not related to the value-added. 

**Then, note that:**

- The `olm.maxOpenShiftVersion` was added for the scenarios where the bundles were using APIs removed on 4.9 so that this property allows block cluster upgrades. More info: https://github.com/redhat-openshift-ecosystem/community-operators-prod/discussions/138

- This feature is only useful for Openshift 
- Solutions distributed on this catalog are NOT provided and used in OCP or OKD by default (You can check [here](https://docs.openshift.com/container-platform/4.6/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs) the sources used to built OCP catalog)
- We should only use `olm.maxOpenShiftVersion` when we know that our version will NOT work on the future version. 
- The annotation was added in the bundles where we could check that were using removed APIs on 4.9/k8s 1.22 and could NOT work on OCP 4.9 at all. The process to set these values automatically was NOT done in this repository.

In this way, why are you adding this annotation here?
What is your intention with?
Why did you set the value as v4.10?

c/c @J0zi @mvalarh 

/hold